posthog/hogql/bytecode/test/test_bytecode_execute.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+
+from posthog.hogql.bytecode.create import create_bytecode
+from posthog.hogql.bytecode.execute import execute_bytecode, get_nested_value
+from posthog.hogql.parser import parse_expr
+from posthog.test.base import BaseTest
+
+
+class TestBytecodeExecute(BaseTest):
+    def _run(self, expr: str) -> Any:
+        fields = {
+            "properties": {"foo": "bar"},
+        }
+        return execute_bytecode(create_bytecode(parse_expr(expr)), fields)
+
+    def test_bytecode_create(self):
+        self.assertEqual(self._run("1 + 2"), 3)
+        self.assertEqual(self._run("1 - 2"), -1)
```

Comment:
Are unhappy/weird paths tested too somewhere, e.g. `(1 + )` or `not true true` or `('a' in 'train') in 'car'`?

Reply:
now they are